.github/workflows/node.js.yml

```diff
@@ -0,0 +1,26 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
```

Comment:
Try removing `cache: "npm"` ? I think the CI fails cause of it.
Also, please use pinned version for GA (you can run `npx node-ga-updater -u` it will update it)